bin/fetch-credhub-credentials.rb

```diff
@@ -6,6 +6,11 @@
 env_dir = ARGV[1]
 file_dir = ARGV[2]
 
+if ENV["CREDHUB_USE_CF_INSTANCE_IDENTITY"] == "true"
```

Comment:
```suggestion
if not ENV.key? 'CREDHUB_CLIENT' or ENV.key? 'CREDHUB_SECRET'
```
I think make this the default auth method since it is the only one that doesn't require any external data. This leaves it really clean when you are using cert auth since you just include `CREDHUB_SERVER` and `CREDHUB_CA_CERT`.